lib/hdk_semantic_indexes/zome/src/lib.rs

```diff
@@ -344,13 +331,20 @@ fn create_dest_indexes<'a, A, B, S>(
     link_tag: &'a S,
     link_tag_reciprocal: &'a S,
 ) -> Box<dyn for<'r> Fn(&B) -> Vec<RecordAPIResult<ActionHash>> + 'a>
-    where S: 'a + AsRef<[u8]> + ?Sized + std::fmt::Debug,
+    where S: 'a + AsRef<[u8]> + ?Sized,
         A: DnaAddressable<EntryHash>,
         B: DnaAddressable<EntryHash>,
 {
     Box::new(move |dest| {
         match create_index(source, dest, link_tag, link_tag_reciprocal) {
-            Ok(created) => created,
+            Ok(created) => created.iter().cloned()
```

Comment:
If you can leave some developer facing code comments on this block of code then I'd be all sold on merging this!

Reply:
I just mean the whole filter map

Reply:
I'll do you some better on code docs